src/main/java/hudson/tasks/junit/TestResultAction.java

```diff
@@ -86,8 +88,14 @@ public TestResultAction(TestResult result, BuildListener listener) {
         this((Run) null, result, listener);
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
The call to `Job.getActions()` cannot be avoided here. Normally it should not be used, but this is one of the exceptions.